actors/builtin/storage_miner/storage_miner_actor.go

```diff
@@ -281,12 +281,12 @@ func (a *StorageMinerActor) ExtendSectorExpiration(rt Runtime, sectorNumber abi.
 
 		sectorInfo, found := st.Sectors[sectorNumber]
 		if !found {
-			rt.AbortStateMsg("Sector not found")
+			rt.Abort(exitcode.ErrNotFound, "Sector not found")
 		}
 
 		extensionLength = newExpiration - sectorInfo.Info.Expiration
 		if extensionLength < 0 {
-			rt.AbortStateMsg("Cannot reduce sector expiration")
+			rt.Abort(exitcode.ErrForbidden, "Cannot reduce sector expiration")
```

Comment:
IllegalArgument? My understanding is that ErrForbidden refers to actors invoking actions that they do not have the right to

Reply:
ah ok. if that;s the case you're right (but a lot of your errForbidden suggestions above are wrong :b)